packages/loopback/lib/application.ts

```diff
@@ -20,8 +20,9 @@ export class Application extends Context {
       }
 
       const ctorFactory = (req: http.ServerRequest, res: http.ServerResponse) => {
-        // TODO(bajtos) Create a new nested/child per-request Context
-        const requestContext = this;
+        const requestContext = new Context(this);
+        requestContext.bind('http.request').to(req);
```

Comment:
I wonder if we should use an even more qualified name as `transport.http.request`. Such namespace/package can help us organize the context. Food for thoughts.